docs/src/app/(public)/layout.tsx

```diff
@@ -10,7 +10,7 @@ export default async function RootLayout({
   const navigationMap = await getNavigationMap();
 
   return (
-    <div className="min-h-screen flex flex-col">
+    <div className="flex min-h-screen flex-col text-slate-12 antialiased">
```

Comment:
The `antialiased` class here is responsible for the whole site feeling a bit "thinner".

Uploading antialiased.mov…